meson.build

```diff
@@ -1,6 +1,6 @@
 project('thorvg',
         'cpp',
-        default_options : ['buildtype=debugoptimized', 'b_sanitize=none', 'werror=false', 'optimization=3', 'cpp_std=c++14', 'strip=true'],
+        default_options : ['buildtype=debugoptimized', 'b_sanitize=none', 'werror=false','c_args=-Wdouble-promotion', 'cpp_args=-Wdouble-promotion', 'optimization=3', 'cpp_std=c++14', 'strip=true'],
```

Comment:
We may need to further review whether to add 'double-promotion.' Alongside that, I'd like to suggest adding these arguments in `src/meson.build`. Also, since ThorVG is primarily a C++ project, I think `c_args` may not be strictly necessary.

Reply:
Yeah I used this locally to not miss any warnings. Just put them into the PR because having a strict set of rules is quite useful in CI I think

Reply:
Yeah I used this locally to not miss any warnings. Just put them into the PR because having a strict set of rules is quite useful in CI I think.

Reply:
I'm not sure if adding only this setting here will be of practical help to the CI. I think, it might be better to aim for consistency.